src/Control/Monad/Logic/Sequence/Morph.hs

```diff
@@ -0,0 +1,9 @@
+module Control.Monad.Logic.Sequence.Morph
+  ( hoistUnexposed
+  , hoistPost
+  , hoistPostUnexposed
+  , hoist
```

Comment:
I'm a little uncomfortable about re-exporting the `hoist` method. Should I

1. Export a type-specialized one instead?
2. Rename `hoistUnexposed` to `hoistPreUnexposed` and export a type-specialized `hoist` called `hoistPre`?

I'm leaning towards 2. Note: the `Unexposed` terminology is stolen from `streaming`.

Reply:
I'll be honest. I don't know much about this hoist stuff or how often anyone will need it. Are you uncomfortable because it's it can be unsafe in some context or just trying not to pollute things when people import stuff?